samples/net/secure_mqtt_sensor_actuator/src/device.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __DEVICE_H__
+#define __DEVICE_H__
+
+/** @brief Sensor sample structure */
+struct sensor_sample {
+	const char *unit;
+	int value;
+};
+
+/** @brief Available board LEDs */
+typedef enum {
+	LED_NET, /* Network status LED*/
+	LED_USER /* User LED */
+} led_id_t;
+
+/** @brief LED states */
+typedef enum {
+	LED_OFF,
+	LED_ON
+} led_state_t;
+
+/** @brief Device command consisting of a command string
+ *         and associated handler function pointer
+ */
+typedef struct device_cmd {
```

Comment:
Please don't use typedefs, it's against Zephyr coding guidelines.

Reply:
Done.